modules/util/portal-tools-service-builder-test-service/src/main/java/com/liferay/portal/tools/service/builder/test/object/system/ERCCompanyEntrySystemObjectDefinitionManager.java

```diff
@@ -0,0 +1,178 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.tools.service.builder.test.object.system;
+
+import com.liferay.counter.kernel.service.CounterLocalService;
+import com.liferay.object.constants.ObjectDefinitionConstants;
+import com.liferay.object.field.builder.IntegerObjectFieldBuilder;
+import com.liferay.object.model.ObjectField;
+import com.liferay.object.system.BaseSystemObjectDefinitionManager;
+import com.liferay.object.system.JaxRsApplicationDescriptor;
+import com.liferay.object.system.SystemObjectDefinitionManager;
+import com.liferay.petra.sql.dsl.Column;
+import com.liferay.petra.sql.dsl.Table;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.BaseModel;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.tools.service.builder.test.model.ERCCompanyEntry;
+import com.liferay.portal.tools.service.builder.test.model.ERCCompanyEntryTable;
+import com.liferay.portal.tools.service.builder.test.service.ERCCompanyEntryLocalService;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Gabriel Albuquerque
+ */
+@Component(service = SystemObjectDefinitionManager.class)
+public class ERCCompanyEntrySystemObjectDefinitionManager
+	extends BaseSystemObjectDefinitionManager {
+
+	@Override
+	public long addBaseModel(User user, Map<String, Object> values)
+		throws Exception {
+
+		ERCCompanyEntry ercCompanyEntry =
+			_ercCompanyEntryLocalService.createERCCompanyEntry(
+				_counterLocalService.increment());
+
+		ercCompanyEntry.setExternalReferenceCode(
+			GetterUtil.getString(values.get("externalReferenceCode")));
+		ercCompanyEntry.setColumn1(
+			GetterUtil.getInteger(values.get("column1")));
+
+		ERCCompanyEntry persistedERCCompanyEntry =
```

Comment:
@gabrielwas why did you have to create another variable here? can't we just reuse the existing one?

Reply:
@carolmariaabb good! Thx.